fmc/src/flow/cold_reset.rs

```diff
@@ -0,0 +1,38 @@
+/*++
+
+Licensed under the Apache-2.0 license.
+
+File Name:
+
+    warm_reset.rs
+
+Abstract:
+
+    File contains the implementation of warm reset flow.
+
+--*/
+use crate::fmc_env::FmcEnv;
+use crate::HandOff;
+use caliptra_common::cprintln;
+use caliptra_drivers::CaliptraResult;
+
+/// Warm Reset Flow
+pub struct ColdResetFlow {}
+
+impl ColdResetFlow {
+    /// Execute update reset flow
+    ///
+    /// # Arguments
+    ///
+    /// * `env` - FMC Environment
+    #[inline(never)]
```

Comment:
just curious, why don't we want to inline this?

Reply:
Make it easy for debug.